<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="graphic_widget_member" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.147727 rgba(0, 135, 205, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QCommandLinkButton" name="login_member_bck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Back page</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>801</height>
     </rect>
    </property>
    <widget class="QPushButton" name="login_member_view_finsh_btn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>650</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fnish</string>
     </property>
    </widget>
    <widget class="QLabel" name="video_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="video_label_info">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>471</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="video_label_counter">
     <property name="geometry">
      <rect>
       <x>906</x>
       <y>200</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>login_member_bck</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
